README.md

```diff
@@ -66,21 +66,70 @@ $docker run -p 8443:443 my-modsec
 
 ### Proxying
 
-ModSecurity is often used as a reverse proxy. This allows one to use ModSecurity without modifying the webserver hosting the underlying application (and also protect web servers that modsecurity cannot currently embedd into). To set proxying one must set SETPROXY to True and provide the location of the Proxy via the PROXYLOCATION argument. Note: if you are going to be proxying to a HTTPS site, you'll need to set SETTLS to true also.
+ModSecurity is often used as a reverse proxy. This allows one to use ModSecurity without modifying the webserver hosting the underlying application (and also protect web servers that modsecurity cannot currently embedd into). To set proxying one must set SETPROXY to True and provide the location of the Proxy via the BACKEND environment variable. Note: if you are going to be proxying to a HTTPS site, you'll need to set SETTLS to true also.
 
 ```
-$docker build --build-arg SETPROXY=True --build-arg PROXYLOCATION=http://example.com -t my-modsec .
-$ docker run -p 8080:80 my-modsec
+$docker build --build-arg SETPROXY=True -t my-modsec .
```

Comment:
Should there be a space between `$` and `docker`?

Also, do we need this line at all? We removed the `SETPROXY`, didn't we?

Reply:
Still working on the README